pkg/services/ngalert/api/tooling/post.json

```diff
@@ -3114,6 +3114,7 @@
    "x-go-package": "github.com/prometheus/alertmanager/api/v2/models"
   },
   "alertGroup": {
+   "description": "AlertGroup alert group",
```

Comment:
Reviewers can skip changes in `post.json` and `spec.json` files. We thrash a few keys every time this file gets regenerated. This was not introduced by this PR. There are no other changes to these files.